### PR TITLE
fix: release audit and load

### PR DESCRIPTION
**Pull request by CraigMChen**
#### What this PR does / why we need it:

1. fix relese manage audit
2. use zip file to upload release

#### Which issue(s) this PR fixes:

- Fixes #your-issue_number
- [Erda Cloud Issue Link](paste your link here)

#### Specified Reviewers:

/assign @luobily 

#### ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English | 1. fix relese manage audit; 2. use zip file to upload release |
| 🇨🇳 中文    | 1.修复制品管理的审计问题；2. 使用zip文件上传制品 |

### CODE REVIEW

File: modules/dicehub/endpoints/release.go [hunk 1]
```diff
@@ -480,12 +490,23 @@ func (e *Endpoints) DeleteReleases(ctx context.Context, r *http.Request, vars ma
...
+				templateName = string(apistructs.DeleteProjectReleaseTemplate)
+			}
+			auditCtx["version"] = releases[0].Version
+			auditCtx["releaseId"] = releases[0].ReleaseID
+		} else {
+			templateName = string(apistructs.BatchDeleteReleaseTemplate)
+			auditCtx["versionList"] = strings.Join(versionList, ", ")
+		}
+		if err := e.audit(r, orgID, releasesDeleteRequest.ProjectID, identityInfo.UserID, templateName, auditCtx); err != nil {
+			logrus.Errorf("failed to create audit event for deleting release, %v", err)
```
Comment (luobily) [inline modules/dicehub/endpoints/release.go:509]: warp a func

Comment (luobily) [reply modules/dicehub/endpoints/release.go:509]: used many times

Comment (CraigMChen) [reply modules/dicehub/endpoints/release.go:536]: A little duplication is often better than a little dependency.

**Pull request merged**